lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -219,6 +240,22 @@ click_dropdown_field:
   - selector:
       xpath: //*[(.='<dropdown_field>')]/following-sibling::*//button[contains(@class, 'dropdown__toggle')]
     op: click
+click_first_item_in_resource_list:
+  elements:
+  - selector:
+      xpath: //td[1]//span[@class='co-resource-item']/a
+    timeout: 30
```

Comment:
Sometimes the resources in shared env are stuck to list for some reason, need to wait for a few seconds here.

Reply:
Before clicking, we need wait until page is not loading, we need use another action not simply adding timeout here

Reply:
ok, updated and passed.